inlang/source-code/cross-sell/cross-sell-sherlock/src/recommendation/recommendation.ts

```diff
@@ -0,0 +1,60 @@
+import type { NodeishFilesystem } from "@lix-js/fs"
+import { joinPath } from "./utils/joinPath.js"
+import { isExtensionsJson } from "./utils/isExtensionJson.js"
+import { parse, type CommentJSONValue, stringify } from "comment-json"
+import type { ExtensionsJson } from "./utils/types.js"
+
+export async function addRecommendationToWorkspace(
+	fs: NodeishFilesystem,
+	workingDirectory?: string
+): Promise<void> {
+	const vscodeFolderPath = joinPath(workingDirectory ?? "", ".vscode")
```

Comment:
Should we use `./.vscode` instead of just `.vscode` if no working directory is given? That's how it's currently done in paraglide init

Reply:
done in https://github.com/opral/monorepo/pull/2360/commits/0995046e0633c70b5a85d440419e682950d3770c